drivers/wifi/winc1500/wifi_winc1500_nm_bus_wrapper.c

```diff
@@ -87,6 +94,37 @@ static s8_t spi_rw(u8_t *mosi, u8_t *miso, u16_t size)
 
 #endif
 
+struct winc1500_gpio_configuration *winc1500_configure_gpios(void)
+{
+	struct device *gpio_en, *gpio_irq, *gpio_reset;
+
+	gpio_en = device_get_binding(
+		DT_INST_0_ATMEL_WINC1500_ENABLE_GPIOS_CONTROLLER);
+	gpio_irq = device_get_binding(
+		DT_INST_0_ATMEL_WINC1500_IRQ_GPIOS_CONTROLLER);
+	gpio_reset = device_get_binding(
+		DT_INST_0_ATMEL_WINC1500_RESET_GPIOS_CONTROLLER);
+
+	gpio_pin_configure(gpio_en,
+			   winc1500_gpios[WINC1500_GPIO_IDX_CHIP_EN].pin,
+			   GPIO_OUTPUT |
```

Comment:
The motivation for using `GPIO_OUTPUT_INACTIVE` is that it clearly specifies the initial condition.  Without it we have to rely on some other code somewhere initializing the signal to a reasonable state.  Presumably that's being done, but it's not being done here, so not providing an initial state makes it harder to understand what the driver's doing.

Reply:
sure, but that again requires driver logic change which I am not willing to do (mostly because I cannot test). So let's stick to "port it to what logic was being applied".

Reply:
That's below my normal minimal bar for Zephyr quality given the risk of the change is low and as-is a known cheap technical debt is left in place.

But since we need to get this in today if you don't want to finish it up I can live with it.  @mnkp may or may not insist; I'll wait to approve until there's more feedback.  We'll discuss any outstanding issues in the API telecon and resolve them there.

Reply:
I also believe we should initialize the output pin to the known value. Otherwise, by choice, we will get driver dependent 'random' value. Some of the Zephyr drivers winc1500 is used with may initialize the pint to low the other to high. Even if we can't test, setting the pin to what we believe is a proper value is better than leaving it to the driver. We should use however physical level flags, e.g. `GPIO_OUTPUT_LOW`.